holoviews/tests/operation/test_datashader.py

```diff
@@ -997,6 +997,67 @@ def test_regrid_zero_range(self):
         expected = Image(np.zeros((0, 0)), bounds=(0, 0, 0, 0), xdensity=1, ydensity=1)
         self.assertEqual(regridded, expected)
 
+    # None, False, and 'nearest' are expected to return the same Image values
+    def test_regrid_interpolation_none(self):
+        img = Image(([0.5, 1.5], [0.5, 1.5], [[0, 1], [2, 3]]))
+        regridded = regrid(img, width=4, height=4, upsample=True, interpolation=None, dynamic=False)
+        expected = Image(([0.25, 0.75, 1.25, 1.75], [0.25, 0.75, 1.25, 1.75],
+                          [[0, 0, 1, 1],
+                           [0, 0, 1, 1],
+                           [2, 2, 3, 3],
+                           [2, 2, 3, 3]]))
+        self.assertEqual(regridded, expected)
+
+    def test_regrid_interpolation_false(self):
+        img = Image(([0.5, 1.5], [0.5, 1.5], [[0, 1], [2, 3]]))
+        regridded = regrid(img, width=4, height=4, upsample=True, interpolation=False, dynamic=False)
+        expected = Image(([0.25, 0.75, 1.25, 1.75], [0.25, 0.75, 1.25, 1.75],
+                          [[0, 0, 1, 1],
+                           [0, 0, 1, 1],
+                           [2, 2, 3, 3],
+                           [2, 2, 3, 3]]))
+        self.assertEqual(regridded, expected)
+
+    def test_regrid_interpolation_nearest(self):
+        img = Image(([0.5, 1.5], [0.5, 1.5], [[0, 1], [2, 3]]))
+        regridded = regrid(img, width=4, height=4, upsample=True, interpolation='nearest', dynamic=False)
+        expected = Image(([0.25, 0.75, 1.25, 1.75], [0.25, 0.75, 1.25, 1.75],
+                          [[0, 0, 1, 1],
+                           [0, 0, 1, 1],
+                           [2, 2, 3, 3],
+                           [2, 2, 3, 3]]))
+        self.assertEqual(regridded, expected)
```

Comment:
```suggestion
    # None, False, and 'nearest' are expected to return the same Image values
    # TODO: @pytest.mark.parametrize("interpolation", [None, False, "nearest"])
    def test_regrid_interpolation_nearest(self):
        for interpolation in [None, False, "nearest"]:
            img = Image(([0.5, 1.5], [0.5, 1.5], [[0, 1], [2, 3]]))
            regridded = regrid(img, width=4, height=4, upsample=True, interpolation=interpolation, dynamic=False)
            expected = Image(([0.25, 0.75, 1.25, 1.75], [0.25, 0.75, 1.25, 1.75],
                              [[0, 0, 1, 1],
                               [0, 0, 1, 1],
                               [2, 2, 3, 3],
                               [2, 2, 3, 3]]))
            self.assertEqual(regridded, expected)
```

I think we should do this, easier to read. Same with the others.